src/AKCore/Scripts/VueComponents/Upcoming/UpcomingListItem.vue

```diff
@@ -76,6 +76,12 @@
           {{ t("not-coming", "common") }}
         </p>
       </template>
+      <div v-if="event.disabled">   
```

Comment:
Borde det inte gå att länka in till spelningen så att man kan se de som anmälde sig innan den blev disablad.

Och om det inte ska gå behöver du gömma detta även i calendervyn.  Den koden ligger i:
Scripts\VueComponents\Upcoming\EventInfoModal.vue

Reply:
Kanske kan man låta felmeddelandet vara kvar men fortfarande ha länk?

Reply:
Jo det ska fortfarande gå att se vilka som anmält sig! Bra fångat, jag har fixat detta nu.